calico/_includes/charts/tigera-operator/Chart.yaml

```
@@ -1,6 +1,12 @@
 apiVersion: v2
 name: tigera-operator
 description: Installs the Tigera operator for Calico
-
+home: https://www.tigera.io/
```

Perhaps this might be a more useful link here?
```suggestion
home: https://projectcalico.docs.tigera.io/about/about-calico
```